### PR TITLE
add support for apps without yaml

### PR DESCRIPTION
**Pull request by Zorgino**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code compiles without warnings (code quality chek)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

File: src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs [hunk 1]
```diff
@@ -76,6 +72,20 @@ public IEnumerable<ApplicationContext> InstanceDaemonApps(IServiceProvider servi
...
+
+            var appTypesWithoutYamlConfig = _loadedDaemonApps
+                    .Where(appType => !result.Select(x => x.ApplicationInstance.GetType()).Contains(appType));
+            
+            foreach (var appType in appTypesWithoutYamlConfig)
+            {
+                var appId = appType
+                        .GetCustomAttributes(typeof(NetDaemonAppAttribute), true)
+                        .Cast<NetDaemonAppAttribute>()
+                        .FirstOrDefault()?.Id ?? appType.Name;
```
Comment (FrankBakkerNl) [inline src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: GetCustomAttribute<NetDaemonAppAttribute>()?.Id ?? appType.Name;

Comment (FrankBakkerNl) [reply src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: I ment the generic overload of GetCustomAttribute but the &lt; and &gt; are removed

Comment (FrankBakkerNl) [reply src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: Because the attribute is only allowed once you gan use the singulat GetCustomAttribute

Comment (Zorgino) [reply src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]:
```suggestion
                        var appId = appType.GetCustomAttribute<NetDaemonAppAttribute>()?.Id ?? appType.Name;
```
is this viable?

Comment (Zorgino) [inline src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: @FrankBakkerNl , is this viable?
```suggestion
                var appId = appType.GetCustomAttribute<NetDaemonAppAttribute>()?.Id ?? appType.Name;
```

Comment (FrankBakkerNl) [reply src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: Yea, that is what I ment it is basically the same as you had before, just en bit shorter and eaaier to read

**Review by FrankBakkerNl — APPROVED** [review]

File: src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs (hunk 1 above)
Comment (Zorgino) [reply src/Daemon/NetDaemon.Daemon/Daemon/CodeManager.cs:84]: it sure is!

**Pull request merged**